q2_feature_classifier/tests/test_classifier.py

```diff
@@ -223,3 +223,16 @@ def test_unassigned_taxa(self):
             id_ = seq.metadata['id']
             assert ref[id_].startswith(classified[id_]) or \
                 classified[id_] == 'Unassigned'
+
+    def test_autotune_reads_per_batch(self):
+        self.assertEqual(
+            _autotune_reads_per_batch(self.seq_path, n_jobs=4), 276)
+
+    def test_autotune_reads_per_batch_disable_if_single_job(self):
+        self.assertEqual(
+            _autotune_reads_per_batch(self.seq_path, n_jobs=1), 500000)
+
+    def test_autotune_reads_per_batch_zero_jobs(self):
```

Comment:
Could you catch this case yourself and raise your own error? I think our error would be better. For example:

```python
if n_jobs == 0:
    raise ValueError("Value other than zero must be specified as number of jobs to run.")
```